### ISSUE TITLE
Accessibility testing

### ISSUE THREAD

**Issue by KrolPolski** [Welhox/Transcendence#35]
Test how well site navigation works with the orca screenreader.

**Comment by KrolPolski** [comment]
I just did an accessibility audit of the site in its current form. It is already largely working with a screen reader (Voiceover from Mac OS).  The settings page works, and can be used with a screen reader alone, but it's not a smooth experience.
1) The announcements of success or failure on saving the image file are not read by the screen reader.
2) When you click the update email or password buttons, the focus moves to the main window instead of the relevant text field.
3) The 2FA toggle switch isn't correctly labeled.

I still need to test with orca.
